compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -283,121 +213,35 @@ public STNode resumeParsing(ParserRuleContext context, Object... args) {
                 return parseOpenBracket();
             case RESOURCE_DEF:
                 return parseResource();
-            case OPTIONAL_SERVICE_NAME:
-                return parseServiceName();
-            case SERVICE_KEYWORD:
-                return parseServiceKeyword();
-            case ON_KEYWORD:
-                return parseOnKeyword();
-            case RESOURCE_KEYWORD:
-                return parseResourceKeyword();
-            case LISTENER_KEYWORD:
-                return parseListenerKeyword();
             case NIL_TYPE_DESCRIPTOR:// following method is only referred in resume parsing
                 return parseNilTypeDescriptor();
-            case COMPOUND_ASSIGNMENT_STMT:// following method is only referred in resume parsing
```

Comment:
`parseNilTypeDescriptor()` above this line also has no usage in other places.

Reply:
yea, will remove that as well.